ale/isd_generate.py

```diff
@@ -89,6 +89,12 @@ def main():
         help="Generate nadir spice pointing, an isd that has pointing directly towards "
              "the center of the target body."
     )
+    parser.add_argument(
+        "-f", "--fix_quaternion",
+        action="store_true",
+        help="There are rare cases when a stream of quaternion flip their signs. This option will "
+             "fix those to be consistent. This takes after a NASA ASP function to do the same. "
+    )
```

Comment:
Are there cases where you wouldn't want this on?

Reply:
Good question. I think @oleg-alexandrov would state this should always be run to fix for all images (that assumes I am doing this correctly). But perhaps. we have a flag that turns this function off instead of on.

Reply:
Weighing in here with a +1 to @oleg-alexandrov's comment below. Always on.